### PR TITLE
Add Cava's Anti Cheat to the supported anti-cheat list

### PR DESCRIPTION
**Pull request by Filocava99**
Add Cava's Anti Cheat to the list

### CODE REVIEW

**Review by hasankayra04 — APPROVED** [review]

File: _docs/geyser/anticheat-compatibility.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ Full Compatibility (Checks Bedrock Players Accurately)
 Partially Compatible (Does not Check/Ignores Bedrock Players)
 
 - [AAC](https://www.spigotmc.org/resources/6442/){:target="_blank"} (Paid) - Last Updated on 19th August 2021
+- [Cava's Anti Cheat](https://www.spigotmc.org/resources/cavas-anti-cheat.102867/){:target="_blank"} - Last Updated on 21th July 2022
```
Comment (rtm516) [inline _docs/geyser/anticheat-compatibility.md:22]:
```suggestion
- [Cava's Anti Cheat](https://www.spigotmc.org/resources/102867/){:target="_blank"} - Last Updated on 21th July 2022
```

**Review by rtm516 — CHANGES_REQUESTED** [review]

**Comment by Filocava99** [comment]
@rtm516 I have applied the requested changes

**Comment by rtm516** [comment]
This looks like a server+client anticheat, are we sure it will even accept Geyser players?

**Comment by Filocava99** [comment]
> This looks like a server+client anticheat, are we sure it will even accept Geyser players?

It does, already been tested by many players. Anyway I am currently not maintaining it anymore so do as you wish

**Comment by onebeastchris** [comment]
I'm not sure we want to include unmaintained anticheats on the list - sorry.

**Pull request closed without merging**